### ISSUE TITLE
Add support for SkinnedMeshRenderer

### ISSUE THREAD

**Issue by yosun** [SixWays/FacePaint#3]
Is there a reason why SkinnedMeshRenderer isn't supported (?) and just MeshRenderer is?

**Comment by SixWays** [comment]
Yes - put simply, MeshRenderer has the additionalVertexStreams property and SkinnedMeshRenderer doesn't. To support SMR requires a completely different approach - namely, making a completely new copy of the mesh in question and altering its vertex colours.

That said, I do keep meaning to implement something along these lines for regular meshes (to improve batching at runtime, at the cost of higher memory usage). The problem for SMRs is that I don't yet know how I'd do this in-editor without a fairly complex system of creating, assigning, un-assigning and destroying copied meshes whenever you save anything.
